images/pgmq-pg/Dockerfile

```
@@ -29,18 +31,20 @@ RUN $HOME/.cargo/bin/cargo pgrx install --pg-config=$(which pg_config)
 
 # install pg_partman
 RUN git clone https://github.com/pgpartman/pg_partman.git && \
-    cd pg_partman && \
-    git checkout v4.7.4 && make && make install
+	cd pg_partman && \
+	git checkout v4.7.4 && make && make install
 
-FROM postgres:15.4-bookworm
+ARG PG_DOCKER_VERSION=$PG_DOCKER_VERSION
```

```suggestion
ARG PG_DOCKER_VERSION=${PG_DOCKER_VERSION}
```
not sure but seems to work here (?)